model/src/main/proto/feedback_reporting.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+
+package model;
+
+import "profile.proto";
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Top-level proto to store feedback reports locally when there is no network connection to upload
+// them to the remote service
+message FeedbackReportingDatabase {
+  repeated FeedbackReport reports_list = 1;
+}
+
+// Corresponds to an entire feedback report with user reported information and collected app,
+// device, and build information.
+message FeedbackReport {
+  // The date the report was created.
+  int64 report_date_ms = 1;
+  FeedbackReportingForm feedback_form = 2;
+  FeedbackReportingDeviceInfo device_info = 3;
+  FeedbackReportingBuildInfo build_info = 4;
+  FeedbackReportingAppInfo app_info = 5;
+}
+
+// Information supplied by the user in the feedback report.
+message FeedbackReportingForm {
```

Comment:
Would ``UserSuppliedInfo`` maybe be clearer? It's a bit odd to call this 'form' since it's not actually a form, just the information collected from one. To some extent, 'form' is a UI-specific detail regarding where this information originates.

Reply:
Done, changed to `UserSuppliedFeedback` let me know what you think.

Reply:
Much clearer--thanks!